site/content/docs/4.3/examples/checkout/form-validation.js

```diff
@@ -2,20 +2,18 @@
 (function () {
   'use strict'
 
-  window.addEventListener('load', function () {
-    // Fetch all the forms we want to apply custom Bootstrap validation styles to
-    var forms = document.getElementsByClassName('needs-validation')
+  // Fetch all the forms we want to apply custom Bootstrap validation styles to
+  var forms = document.getElementsByClassName('needs-validation')
 
-    // Loop over them and prevent submission
-    Array.prototype.filter.call(forms, function (form) {
-      form.addEventListener('submit', function (event) {
-        if (form.checkValidity() === false) {
-          event.preventDefault()
-          event.stopPropagation()
-        }
+  // Loop over them and prevent submission
+  Array.prototype.filter.call(forms, function (form) {
```

Comment:
Can you remove filter and use `forEach` instead ? There is no filtering inside...